content/en/docs/tasks/debug-application-cluster/audit.md

```diff
@@ -154,16 +154,14 @@ volumeMounts:
 finally the hostPath:
 
 ```
-- name: audit
-  hostPath:
+- hostPath:
     path: /etc/kubernetes/audit-policy.yaml
     type: File
-
-- name: audit-log
-  hostPath:
+  name: audit
+- hostPath:
     path: /var/log/audit.log
     type: FileOrCreate
-    
+  name: audit-log   
```

Comment:
Not quite sure what's wrong with the old version, the order doesn't matter.

Reply:
Hi @soltysh I was having some issues with my kube-apiserver because of this after the changes and it is also the way that other volumes are setup on the kube-apiserver.yaml file. If not really a problem please close this issue. Thanks!

From the kube-apiserver.yaml file:
volumes:
  - hostPath:
      path: /etc/ssl/certs
      type: DirectoryOrCreate
    name: ca-certs
  - hostPath:
      path: /etc/ca-certificates
      type: DirectoryOrCreate
    name: etc-ca-certificates
  - hostPath:
      path: /etc/kubernetes/pki
      type: DirectoryOrCreate
    name: k8s-certs
  - hostPath:
      path: /usr/local/share/ca-certificates
      type: DirectoryOrCreate
    name: usr-local-share-ca-certificates
  - hostPath:
      path: /usr/share/ca-certificates
      type: DirectoryOrCreate
    name: usr-share-ca-certificates
  - hostPath:
      path: /etc/kubernetes/audit-policy.yaml
      type: File
    name: audit  
  - hostPath:
      path: /var/log/audit.log
      type: FileOrCreate
    name: audit-log